### PR TITLE
* Adding in Forbidden Power elements. (Fixes #192).

### LINKED ISSUE #192: Add forbidden power endless spells
Forbidden power has added four new endless spells. 
- soulscream bridge 
- horrorghast 
- lauchon the soul seeker 
- shards of valagharr

### PR DESCRIPTION
**Pull request by JohnSchramm**
* Merged malign sorcery files into generic army.ts

* Added placeholder files for various modifications relating to realm features. (#228).

* Added generic command abilities and triumphs. (#229)

* Updated components and types files with starting support for above.

* Removed malign sorcery army as it is now rolled into generic.

### CODE REVIEW

File: src/army/generic/realmscape_commands.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  SHOOTING_PHASE,
+  START_OF_COMBAT_PHASE,
+  START_OF_HERO_PHASE,
+  START_OF_MOVEMENT_PHASE,
+  START_OF_ROUND,
+} from 'types/phases'
+import { TCommands } from 'types/army'
+import { AQSHY, CHAMON, GHUR, GHYRAN, HYSH, SHYISH, STYGXX, ULGU } from 'types/realmscapes'
+
+// Realm specific command abilities.
```
Comment (daviseford) [inline src/army/generic/realmscape_commands.ts]: TIL that there are realm-specific command abilities 😮

File: src/army/generic/realmscape_features.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ import {
   START_OF_HERO_PHASE,
   START_OF_MOVEMENT_PHASE,
 } from 'types/phases'
-import { SHYISH, AQSHY, CHAMON, GHUR, GHYRAN, HYSH, ULGU } from 'types/realmscapes'
+import { AQSHY, CHAMON, GHUR, GHYRAN, HYSH, SHYISH, STYGXX, ULGU } from 'types/realmscapes'
```
Comment (daviseford) [inline src/army/generic/realmscape_features.ts]: Is `STYGXX` a new realm from forbidden power? Could GW please make pronouncable realms 🤣

File: src/army/generic/spells.ts [hunk 3]
```diff
@@ -19,7 +20,12 @@ const Spells: TSpells = [
       {
         name: `Mystic Shield`,
         desc: `Casting value of 6. If successfully cast, pick a friendly unit within 18" of the caster that is visible to them. Re-roll save rolls of 1 for that unit until your next hero phase.`,
-        when: [HERO_PHASE, COMBAT_PHASE, SHOOTING_PHASE],
+        when: [HERO_PHASE],
+      },
+      {
+        name: `Mystic Shield`,
+        desc: `If active, pick a friendly unit within 18" of the caster that is visible to them. Re-roll save rolls of 1 for that unit until your next hero phase.`,
```
Comment (daviseford) [inline src/army/generic/spells.ts]: Change to `If successfully cast, re-roll save rolls...`

File: src/utils/getArmy.ts [hunk 4]
```diff
@@ -100,7 +105,9 @@ const modifySpells = (spells: TSpells, realmscape: TRealms | null): TSpells => {
 }
 
 const modifyScenery = (scenery: TScenery): TScenery => {
-  return sortBy(scenery, 'name').map(s => ({ ...s, scenery: true }))
+  return sortBy(scenery, 'name')
+    .concat(sortBy(GenericScenery, 'name'))
+    .map(s => ({ ...s, scenery: true }))
```
Comment (daviseford) [inline src/utils/getArmy.ts]: 🔥

File: src/army/generic/realmscape_features.ts (hunk 2 above)
Comment (JohnSchramm) [reply src/army/generic/realmscape_features.ts]: Yeah they love their goofy naming conventions!

File: src/army/generic/realmscape_commands.ts (hunk 1 above)
Comment (JohnSchramm) [reply src/army/generic/realmscape_commands.ts]: Yeah they mixed them in with realm-specific spells active wherever the battle takes place.

**Pull request merged**